### PR TITLE
docs(apps/code-examples): add key info spec example

### PR DESCRIPTION
**Pull request by Blackbaud-CoreyArcher**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/517/storybook/)

Component Storybooks:

- _(no component storybooks affected in this pr)_ 

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/517/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/517/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/517/playground/)

File: apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { TestBed } from '@angular/core/testing';
+import { SkyKeyInfoHarness } from '@skyux/indicators/testing';
+
+import { KeyInfoDemoComponent } from './key-info-demo.component';
+import { KeyInfoDemoModule } from './key-info-demo.module';
+
+describe('Basic key info', () => {
+  async function setupTest(options?: { layout?: 'vertical' | 'horizontal' }) {
```
Comment (Blackbaud-TrevorBurch) [inline apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.spec.ts]: Can we type this with the new `SkyKeyInfoLayoutType`?

File: apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [KeyInfoDemoModule],
+    });
+  });
+
+  it('should display a vertical key info', async () => {
+    const { keyInfoHarness, fixture } = await setupTest({ layout: 'vertical' });
+    fixture.detectChanges();
```
Comment (Blackbaud-TrevorBurch) [inline apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.spec.ts]: Are these `fixture.detectChanges` needed? Just verifying as I know we were seeing that the harness loader seemed to be doing that by default at one point.

File: apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<sky-key-info [layout]="layout">
+<sky-key-info [layout]="layout" data-sky-id="key-info-demo">
   <sky-key-info-value> 575 </sky-key-info-value>
```
Comment (Blackbaud-TrevorBurch) [inline apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.html]: I know this is splitting hairs - but should we do some sort of logic like we did with "Alert" where maybe we have a `memberCount` variable that then controls the layout. So under 100 is horizontal or something and then we can change that variable and make sure the test updates correctly?

File: apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.ts [hunk 4]
```diff
@@ -1,9 +1,18 @@
...
+import { Component, Input } from '@angular/core';
 
 @Component({
   selector: 'app-key-info-demo',
   templateUrl: './key-info-demo.component.html',
 })
 export class KeyInfoDemoComponent {
-  public layout = 'vertical';
+  @Input()
+  public set layout(value: 'vertical' | 'horizontal') {
```
Comment (Blackbaud-TrevorBurch) [inline apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.ts]: Same comment about typing this with the new type.

Comment (Blackbaud-CoreyArcher) [reply apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.ts]: changed to type

File: apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.html (hunk 3 above)
Comment (Blackbaud-CoreyArcher) [reply apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.html]: updated tests

File: apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.spec.ts (hunk 2 above)
Comment (Blackbaud-CoreyArcher) [reply apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.spec.ts]: they apparently aren't needed. removed

File: apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.spec.ts (hunk 1 above)
Comment (Blackbaud-CoreyArcher) [reply apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.spec.ts]: after updating the tests, the `SkyKeyInfoLayoutType` is no longer exposed to the consumer

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**